test/py/test_dirty_pages.py

```diff
@@ -271,7 +283,13 @@ def test_dirty_pages_get_unmodified():
     assert br.bitmap.size == 8
 
 
-def get_dirty_page_bitmap():
+def get_dirty_page_bitmap_result(resp):
+    dirty_pages, resp = vfio_user_dirty_pages.pop_from_buffer(resp)
```

Comment:
nit: since you're changing this code anyway, we don't seem to use `dirty_pages` and `br` so can we replace them with `_`?